pages/index.js

```diff
@@ -1,7 +1,21 @@
-export default function HomePage() {
+import Header from "../components/Header/Header";
+import Image from "next/image";
+import Counter from "../components/Counter/Counter";
+import StyledDiv from "../components/StyledDiv";
+
+export default function HomePage({ countries, countriesInfo }) {
   return (
     <>
-      <p>Homepage will be added in next US</p>
+      <Header headline="globehopper" />
+      <StyledDiv>
+        <Image
+          src="/../public/assets/worldmap.jpg"
```

Comment:
The image didn't show when I looked at the preview. Maybe this slash could be the problem?

```suggestion
          src="../public/assets/worldmap.jpg"
```